src/NServiceBus.AcceptanceTests/DelayedDelivery/TimeoutManager/When_timeout_dispatch_fails_on_timeout_data_removal.cs

```diff
@@ -123,11 +122,11 @@ public Task RemoveTimeoutBy(Guid sagaId, ContextBag context)
                 public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice)
                 {
                     var timeouts = timeoutData != null
-                        ? new List<TimeoutsChunk.Timeout>
+                        ? new TimeoutsChunk.Timeout[1]
```

Comment:
> I thought by default the have a size by the power of two or something but I don't remember

that's the lists's behavior? If you specify the array with a size of [2], the second item will be null, whereas the list only shows one item even if you initialized with with size [2].